src/main/kotlin/no/nav/familie/tilbake/oppgave/OppgavePrioritetService.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.familie.tilbake.oppgave
+
+import no.nav.familie.kontrakter.felles.oppgave.Oppgave
+import no.nav.familie.kontrakter.felles.oppgave.OppgavePrioritet
+import no.nav.familie.tilbake.config.FeatureToggleConfig
+import no.nav.familie.tilbake.config.FeatureToggleService
+import no.nav.familie.tilbake.kravgrunnlag.KravgrunnlagRepository
+import no.nav.familie.tilbake.kravgrunnlag.domain.Klassetype
+import no.nav.familie.tilbake.kravgrunnlag.domain.Kravgrunnlag431
+import org.springframework.stereotype.Service
+import java.math.BigDecimal
+import java.util.UUID
+
+@Service
+class OppgavePrioritetService(
+    private val kravgrunnlagRepository: KravgrunnlagRepository,
+    private val featureToggleService: FeatureToggleService
+) {
+
+    fun utledOppgaveprioritet(behandlingId: UUID, oppgave: Oppgave? = null): OppgavePrioritet {
+        val finnesKravgrunnlag = kravgrunnlagRepository.existsByBehandlingIdAndAktivTrue(behandlingId)
```

Comment:
Når vi kaller på opprett tilbakekreving fra iverksett, finnes det allerede ett kravgrunnlag då? (Sender vi med kravgrunnlag?) Eller er det noe som senere hentes fra økonomi?

Reply:
Det er tre måter tilbakekreving opprettes:

- I behandlingsflyt (via iverksett)
- Manuell opprettelse fra behandlingsoversikten

Reply:
Ok, ja, så etter 1-2 dager så settes prioritet, det burde vel gå fint :)

Hvis det er viktig at få ned prioriteten fra når man oppretter oppgaven (hva tenker de funksjonelle?) så kan man ju sende med beløpet fra ef-sak (det sendes kun med nå hvis det er tilbakekreving med varsel)

Reply:
Så litt på det, og dét vil gjelde for ca 1/3 av sakene, så usikker på om det gir stor verdi, eller bare skaper forvirring... Saken er jo uansett "satt på vent" når den står på normal første gang.

Reply:
Hva er flytet her, når man oppretter en behandling via iverksett, settes venteårsaken til `VENT_PÅ_TILBAKEKREVINGSGRUNNLAG`eller når gjøres det?

Reply:
Ja, den settes enten til "VENT_PÅ_TILBAKEKREVINGSGRUNNLAG" eller "VENT_PÅ_BRUKERTILBAKEMELDING"